nautobot/extras/management/commands/runjob.py

```diff
@@ -62,13 +62,7 @@ def handle(self, *args, **options):
         self.stdout.write(f"[{timezone.now():%H:%M:%S}] Running {job_class.class_path}...")
 
         if options["local"]:
-            job_result = JobResult.execute_job(
-                job_model=job_model,
-                user=user,
-                profile=options["profile"],
-                **data,
-            )
-
+            job_result = JobResult.execute_job(job_model, user, profile=options["profile"], **data)
         else:
             job_result = JobResult.enqueue_job(job_model, user, profile=options["profile"], **data)
```

Comment:
Any reason not to stick to specifying with kwargs?

Reply:
The signature for both of these methods is the same so I just wanted to make them consistent here